addons/folo/common/src/main/java/org/commonjava/indy/folo/data/FoloAccessor.java

```diff
@@ -0,0 +1,19 @@
+package org.commonjava.indy.folo.data;
+
+
+import com.datastax.driver.mapping.Result;
+import com.datastax.driver.mapping.annotations.Accessor;
+import com.datastax.driver.mapping.annotations.Param;
+import com.datastax.driver.mapping.annotations.Query;
+
+@Accessor
+public interface FoloAccessor {
+
+    @Query("SELECT * FROM folo.records WHERE path=?")
```

Comment:
Ref the config both for keyspace and table name.

Reply:
This accessor is not. doing. anything, removed.